src/test/groovy/graphql/schema/idl/SchemaTypeCheckerTest.groovy

```diff
@@ -1369,4 +1396,129 @@ class SchemaTypeCheckerTest extends Specification {
         result.isEmpty()
     }
 
+    @Unroll
+    @SuppressWarnings("GroovyPointlessBoolean")
+    def "directive definition allowed argument type '#allowedArgType' does not match argument value '#argValue'"(
+            String allowedArgType, String argValue, String detailedMessage) {
```

Comment:
For the record I dont think you need to declare these as paremters to the method.  Groovy seems happy to just take them as implicit variables by name

Now whether this is "better groovy" I dont know but the other tests that use a where: construct dont do this

Reply:
wow, you're right! I was doing it mechanically but method arguments were never mandatory. It was only to make earlier versions of IntelliJ happy. 

From the Spock docs:

> 1. The idea behind allowing method parameters is to enable better IDE support. However, recent versions of IntelliJ IDEA recognize data variables automatically, and even infer their types from the values contained in the data table.
[1.1 Doc Link](http://spockframework.org/spock/docs/1.1/data_driven_testing.html#_syntactic_variations)